kv/txn_coord_sender.go

```diff
@@ -484,18 +446,16 @@ func (tc *TxnCoordSender) sendOne(ctx context.Context, call proto.Call) {
 						firstUpdateNanos: tc.clock.PhysicalNow(),
 						lastUpdateNanos:  tc.clock.PhysicalNow(),
 						timeoutDuration:  tc.clientTimeout,
-						txnEnd:           make(chan []proto.Key, 1),
+						txnEnd:           make(chan []proto.Key),
```

Comment:
Doesn't look like we're using a value here any more, so `txnEnd` can be changed to a `chan struct{}`.